getReportResults.js

```diff
@@ -0,0 +1,21 @@
+function solution(id_list, report, k) {
+  
+  const uniqueReports = [...new Set(report)]
```

Comment:
- Set으로 중복 신고 제거 후 배열로 변환
- 아래처럼도 쓸수 있음. 
```suggestion
  const uniqueReports = Array.from(new Set(report))
```

Reply:
AI say)

> 추천:
> 가독성을 중시한다면 Array.from(new Set(report))가 좋습니다.
> 간결함을 중시하고, 최신 문법에 익숙하다면 [...new Set(report)]가 더 적합할 수 있습니다.